src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -369,7 +369,7 @@ impl Step for RustAnalyzer {
 
         // We don't need to build the whole Rust Analyzer for the proc-macro-srv test suite,
         // but we do need the standard library to be present.
-        builder.ensure(compile::Std::new(compiler, host));
+        builder.ensure(compile::Rustc::new(compiler, host));
```

Comment:
Like this? Or `target`? Or `builder`? :sweat_smile:

Reply:
this seems fine